server.R

```diff
@@ -9,40 +9,14 @@
 function(input, output, session)
 {
 
-  showModal(
-    modalDialog(
-      title = "Welcome to stanpumpR",
-      p(
-        "stanpumpR, derived from the original STANPUMP program developed at
-        Stanford University,  performs pharmacokinetic simulations
-        based on mathematical models published in the peer-reviewed
-        literature. stanpumpR is intended to help clinicians and investigators
-        better understand the mathematical implications of published models.
-        stanpumpR is only an advisory program. How these models are applied to
-        individual patients is a matter of clinical judgment by the health care
-        provider."
-      ),
-      tags$button(
-        type = "button",
-        class = "btn btn-warning",
-        `data-dismiss` = "modal",
-        "OK"
-      ),
-      footer = NULL,
-      easyClose = TRUE,
-      fade=TRUE,
-      size="s"
-    )
-  )
+  showIntroModal()
```

Comment:
This function has side effects in the UI. Should the direct UI code stay in server.R? I could see this being `showModal(introModal())`. YMMV

Reply:
It's a bit strange and unintuitive, but the current way is the right way. I know it looks strange, but modals are one of shiny's quirks